src/components/com_tjcertificate/administrator/includes/tjcertificate.php

```diff
@@ -130,4 +130,18 @@ public static function loadClass($className)
 
 		return self::$loadedClass[$className];
 	}
+
+	/**
+	 * Initializes the css, js and necessary dependencies
+	 *
+	 * @param   string  $location  The location where the assets needs to load
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public static function init()
+	{
+		self::Language()->siteLanguageConstant();
```

Comment:
I think this will be used in Admin as well...So change the function name to `JsLanguageConstant()`

Reply:
done